fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedStoredFieldsFormatTest.java

```diff
@@ -117,48 +120,39 @@ private static boolean isUsingFDBDirectory() {
         return System.getProperty("tests.directory", "random").equals(TestFDBDirectory.class.getName());
     }
 
+    @Override
```

Comment:
Since these tests change some global settings, would it make sense to have `@AfterAl` added to the end as well?

Reply:
Good idea, but added an override to `tearDown` instead of an `@AfterAll`.